docs/source/en/model_doc/clip.md

```diff
@@ -79,6 +79,12 @@ encode the text and prepare the images. The following example shows how to get t
 >>> probs = logits_per_image.softmax(dim=1)  # we can take the softmax to get the label probabilities
 ```
 
+<Tip>
+
+CLIP supports the usage of efficient algorithms for the attention mechanism through `torch.nn.functional.scaled_dot_product_attention` (SDPA). This can enhance both memory consumption and inference latency. You can expect a speed boost of ~30% when using SDPA, depending on the batch size and the model size. Refer to [this doc](../perf_infer_gpu_one.md) for more details on how to use SDPA. 
```

Comment:
Is this an absolute requirement? I understand this is nice to have but it's an involved effort and not all models supporting SDPA have this information, either. So, I think this is okay to skip for now.

Reply:
Yes, especially for a model as important as CLIP. It really should be added for all models. If it's not tested across batch sizes it's both hard for users to know when to use it and hard for maintainers to answer questions when issues are raised about improvements not being "fast enough".

Reply:
I will add it to keep given the importance of CLIP because many fields (such as diffusion models) benefit from this kind of improvements. Would adding the table just for inference suffice?

(I still find it a little over the requirement for this PR to get approved and merged, though given the fact that many other important models like Phi don’t have such tables)